api/src/main/java/com/datastrato/gravitino/NameIdentifier.java

```diff
@@ -19,6 +19,8 @@ public class NameIdentifier {
 
   private static final Splitter DOT = Splitter.on('.');
 
+  public static final NameIdentifier ROOT = new NameIdentifier();
```

Comment:
This seems will only be used internally, I would suggest not exposing this to the users and add it internally.